workspaces/liferay-testray-workspace/client-extensions/liferay-testray-site-initializer/site-initializer/object-definitions/testray-build.json

```diff
@@ -15,6 +15,17 @@
 			"required": false,
 			"type": "Long"
 		},
+		{
+			"DBType": "Date",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"label": {
+				"en_US": "Date Arquived"
+			},
+			"name": "startDate",
+			"required": false,
+			"type": "Date"
```

Comment:
Is this right? I saw that you added 2 fields with the label "Date Arquived", but one has "startDate" as name and the other "dateArchived". Could you review it?